tests/util_tests/test_shotgun_connect.py

```diff
@@ -407,6 +407,10 @@ class AuthConnectionSettings(ConnectionSettingsTestCases.Impl):
     Tests proxy connection for site and appstore connections.
     """
 
+    def tearDown(self):
+        # Allows deletion of context tmp folder created by TankTestBase.setUpModule (global scope)
+        super(ConnectionSettingsTestCases.Impl, self).tearDown()
```

Comment:
this `ConnectionSettingsTestCases.Impl` defintely deserves some explanatory comment :)